Makefile Windows

```diff
@@ -0,0 +1,34 @@
+# Go parameters
+GOCMD=go
+TEMPL=templ
```

Comment:
This file isn't necessary on windows, `Makefile` isn't usually standard on windows, as long as the `.air_windows.toml` file is right, we shouldn't need this one 😄

Reply:
TL;DR let's remove this from the PR